src/createSliderWithTooltip.jsx

```diff
@@ -1,39 +1,43 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Tooltip from 'rc-tooltip';
-import Handle from './Handle';
+import { Handle } from 'rc-slider';
 
 export default function createSliderWithTooltip(Component) {
   return class ComponentWrapper extends React.Component {
     static propTypes = {
       tipFormatter: PropTypes.func,
       handleStyle: PropTypes.oneOfType([PropTypes.object, PropTypes.arrayOf(PropTypes.object)]),
       tipProps: PropTypes.object,
+      getTooltipContainer: PropTypes.func,
     };
+
     static defaultProps = {
-      tipFormatter(value) { return value; },
+      tipFormatter(value) {
+        return value;
+      },
       handleStyle: [{}],
       tipProps: {},
+      getTooltipContainer: () => document.body
```

Comment:
I think default container should be `node => node.parentNode`.

#20677

Reply:
You right, it should be, I had this idea because it was the default behavior of the createSliderWithTooltip previously